### PR TITLE
Added Button Stories + New Colors from Design

### PR DESCRIPTION
**Pull request by angeljgomezc**
This PR brings the new buttons using the colors from the design system at https://www.figma.com/file/qGlR5rEinlbUgWmbogXunz/Design-System-%2F-Jobs-Board?node-id=119%3A10

The Buttons have a problem, though - they don't show on Storybook

### CODE REVIEW

File: packages/client/.storybook/preview.js [hunk 1]
```diff
@@ -1,6 +1,7 @@
 import React from 'react';
 import { ChakraProvider } from '@chakra-ui/react';
```
Comment (christiananese) [inline packages/client/.storybook/preview.js:3]: you need to import the customt theme here and pass it to the chakraprovider 

``` 

import { theme } from '../styles/theme'; 

....


<ChakraProvider theme={theme}>

```

File: packages/client/stories/Button.stories.tsx [hunk 2]
```diff
@@ -0,0 +1,154 @@
...
+
+  // Sets a click handler to change the label's value
+  const handleOnChange = () => {
+    if (!isPrimary) {
+      setValue('Primary');
+    } else {
+      setValue('Secondary');
+    }
+    setIsPrimary(!isPrimary);
+  };
```
Comment (christiananese) [inline packages/client/stories/Button.stories.tsx:48]: You can move this logic in a wrapper and reuse it for each button type or just remove it entirely I used it just to have an example story when setting storybook up.

**Comment by with-heart** [comment]
I think we should take this opportunity to move away from custom button components entirely and instead utilize [Chakra's component style system](https://chakra-ui.com/docs/theming/component-style) to capture the shared base button styles and implement the stylistic differences between these buttons.

The purpose of this system is to allow complete customization of any components while requiring that users become familiar with only **a single component+props api**:

```tsx
const Current = () => (
  <>
    <ButtonInfo>Click Me</ButtonInfo>
    <ButtonNeutral>Click Me</ButtonNeutral>
    <ButtonWarning>Click Me</ButtonNeutral>
  </>
)

// VS.

const WithComponentStyle = () => (
  <>
    <Button colorScheme="info">Click Me</Button>
    <Button colorScheme="neutral">Click Me</Button>
    <Button colorScheme="warning">Click Me</Button>
  </>
)
```

**Comment by with-heart** [comment]
Actually imo after thinking about it, I think we should break this PR up into two separate PRs:

1. adding + documenting the new colors

  we should build something similar to [Chakra's colors docs](https://chakra-ui.com/docs/theming/theme#colors) for our Storybook
  
  image


2. converting the buttons to use the new colors

File: packages/client/pages/create-profile/index.tsx [hunk 3]
```diff
@@ -28,11 +27,11 @@ import {
...
 import {
-  ButtonBlack,
-  ButtonGreen,
-  ButtonOrange,
-} from '../../styles/ui-components/Chakra-Button';
-import AddAvatar from '../../components/create-profile/AddAvatar';
+  ButtonPrimary,
+  ButtonSuccess,
+  ButtonWarning,
+} from '@/styles/ui-components/Chakra-Button';
```
Comment (frankTurtle) [inline packages/client/pages/create-profile/index.tsx:33]: 🙏

**Review by frankTurtle — COMMENTED** [review]
Having those color themes makes this so much nicer to read!

**Pull request merged**